api/controller/xtra_db_cluster_api.proto

```diff
@@ -67,11 +67,7 @@ message CreateXtraDBClusterRequest {
     }
   ];
   // PMM server public address.
-  string pmm_public_address = 4 [
-    (validator.field) = {
-      string_not_empty: true
-    }
-  ];
+  string pmm_public_address = 4;
```

Comment:
please add TODO or FIXME here

Reply:
What and why we should write in TODO here?